examples/form.rs

```diff
@@ -65,7 +65,7 @@ fn Form<'a>(props: &mut FormProps<'a>, mut hooks: Hooks) -> impl Into<AnyElement
                 }
                 KeyCode::BackTab => focus.set((focus + 3) % 4),
                 KeyCode::Tab => focus.set((focus + 1) % 4),
-                KeyCode::Up if focus != 2 => focus.set((focus + 3) % 4),
+                KeyCode::Up if focus != 0 => focus.set((focus + 3) % 4),
                 KeyCode::Down if focus != 2 => focus.set((focus + 1) % 4),
```

Comment:
With this change, the user would no longer be able to use the up arrow to navigate lines within the multiline text field.

I actually think it might be better to just remove the up/down handling altogether from this example:

```suggestion
```

The UI specifically instructs the user to use tab to cycle between fields, and the up/down logic doesn't really add anything useful to example code.

Reply:
I would suggest deleting the lines here for now and creating an issue describing in detail the behavior you think event handling should have and the new features required to support it.

Reply:
ok

Reply:
Related issue: https://github.com/ccbrown/iocraft/issues/113